docs/team/contributors-jupyter-server.yaml

```
@@ -0,0 +1,62 @@
+# Use ALPHABETICAL (BY LAST NAME) ORDER please :-)
+
+- name: Kevin Bates
+  handle: "@kevin-bates"
+  affiliation: IBM
+  team: red
```

Per my recent comment...
```suggestion
  team: green
```

---

:+1: Thanks, Kevin. Let us know when you're ready to join as Red Team again!